packages/component-library-react/src/css-module/Blockquote.tsx

```diff
@@ -0,0 +1,8 @@
+/**
+ * @license EUPL-1.2
+ * Copyright (c) 2021 Robbert Broersma
+ */
+
+import '../../../../components/blockquote/css/index.scss';
```

Comment:
I think it might be nice if this was referencing the css-component package instead?